buildspec.saas.yml

```diff
@@ -14,6 +14,8 @@ phases:
     commands:
       - VERSION="$(date '+%d%m%Y.%H%M%S')"
       - ls
+      - npm install --unsafe-perm
+      - npm test
```

Comment:
We are running tests on AWS build container and building the application in Dockerfile on alpine. There can be disparity between the builds. Maybe we should run tests in Dockerfile itself?

@SumanthPuram @sivashanmukh What do you guys think?

Reply:
Yea. Makes sense. We should move it to DockerFile

Reply:
Definitely a great option if it's not a big effort.